cmd/snap/cmd_prepare_image.go

```diff
@@ -37,8 +38,10 @@ type cmdPrepareImage struct {
 		Rootdir          string
 	} `positional-args:"yes" required:"yes"`
 
-	Channel    string   `long:"channel" default:"stable"`
-	ExtraSnaps []string `long:"extra-snaps"`
+	Channel string `long:"channel" default:"stable"`
+	// TODO: introduce SnapWithChannel?
```

Comment:
I like the idea, I played around with it a bit http://paste.ubuntu.com/p/xrSCTMZ6QQ/ - but of course fine in a followup.

Reply:
we cannot import snap from asserts, so it would need a bit  more shuffling around